src/__tests__/incidents/Incidents.test.tsx

```diff
@@ -31,28 +31,27 @@ describe('Incidents', () => {
       components: { sidebarCollapsed: false },
     } as any)
 
-    // eslint-disable-next-line react/prop-types
-    const Wrapper: React.FC = ({ children }) => (
+    return render(
       <Provider store={store}>
         <MemoryRouter initialEntries={[path]}>
-          <titleUtil.TitleProvider>{children}</titleUtil.TitleProvider>
+          <titleUtil.TitleProvider>
```

Comment:
This used to be mocked which is why the import is `* as titleUtil`. I think it would read more cleanly if it went back to a normal import.

Reply:
give it a shot 😄